runhouse/rns/folders/folder.py

```diff
@@ -475,7 +475,7 @@ def to_cluster(self, dest_cluster, path=None, mount=False):
                 )
         else:
             raise TypeError(
-                f"`to_cluster` not supported for filesystem type {type(self.system)}"
+                f"`Sending from filesystem type {type(self.system)} is not supported"
```

Comment:
@dongreenberg oh I had a question about this, is sending to a cluster not supported if `self.system` isn't a cluster or file? like if its 's3'

Reply:
I think sending to a data store system like s3 should use `_to_data_store` instead of `_to_cluster`